iceoryx_hoofs/include/iceoryx_hoofs/internal/posix_wrapper/unix_domain_socket.hpp

```diff
@@ -96,43 +93,24 @@ class UnixDomainSocket : public DesignPattern::Creation<UnixDomainSocket, IpcCha
     /// @return received message. In case of an error, IpcChannelError is returned and msg is empty.
     cxx::expected<std::string, IpcChannelError> timedReceive(const units::Duration& timeout) const noexcept;
 
-    /// @brief checks whether the unix domain socket is outdated
-    /// @return true if the unix domain socket is outdated, false otherwise, IpcChannelError if error occured
-    cxx::expected<bool, IpcChannelError> isOutdated() noexcept;
-
   private:
-    /// @brief c'tor
-    /// @param name for the unix domain socket
-    /// @param channel side client or server
-    /// @param maxMsgSize max message size that can be transmitted
-    /// @param maxMsgNumber max messages that can be queued
     UnixDomainSocket(const IpcChannelName_t& name,
                      const IpcChannelSide channelSide,
                      const size_t maxMsgSize = MAX_MESSAGE_SIZE,
                      const uint64_t maxMsgNumber = 10U) noexcept;
 
-    /// @brief c'tor
-    /// @param NoPathPrefix signalling that this constructor does not add a path prefix
```

Comment:
It wouldn't have hurt to keep the documentation... Maybe you can state the intention of the `NoPathPrefix` where it is defined.